server/index.js

```diff
@@ -2,11 +2,115 @@ require('dotenv/config');
 const express = require('express');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
+const jwt = require('jsonwebtoken');
+const cookieParser = require('cookie-parser');
+const pg = require('pg');
+const snoowrap = require('snoowrap');
+const snoostorm = require('snoostorm');
+const ClientError = require('./client-error');
+// eslint-disable-next-line no-unused-vars
```

Comment:
It's not a good habit to disable rules like this. Just delete the unused variable until you need it.